docs/server-setup.md

```diff
@@ -27,7 +27,7 @@ Github link:
 
 1. Download the files above.
 
-1. Create a new folder within the build folder called **Data**. i.e windows: **C:\Build\bin\RelWithDebInfo\Data** or linux: **$HOME/azeroth-server/data/**
+1. Create a new folder within the server folder you created in the previous step, in this case **C:\Server**, called **Data**. i.e **C:\Server\Data**. For linux: **$HOME/azeroth-server/data/**
```

Comment:
You just make it wordy for no reason

Reply:
Since it's a separate page, it makes sense to reference back to the previous step, or at least I think so.